GeofenceDisplay/src/geofence-display.ts

```diff
@@ -2,153 +2,82 @@ export {}
 
 import L from 'leaflet'
 import './geofence-display.css'
+import './rastercoords.js'
 
 const PlantquestGeofenceDisplay = L.Layer.extend({
-  options: {
-    // geofenceLabelPane.style.zIndex = 235
-    // provide geofence data to component using an option
-  },
-
   initialize: function (this: any, options: any) {
-    console.log('PGeoD init')
+    console.log('GeofenceDisplay init function called.')
+    L.Util.setOptions(this, options)
     this._state = {
-      // this._state.GeofenceByID
-      // self.geofence.map[ent.id] = self.config.geofence.prepare(geofence) || geofence
-      //
-      // self.data goes in here
-      // self.data.geofence = ['mock-data-1', 'mock-data-2', 'mock-data-3']
+      zindex: 0,
+      geofenceByID: {},
     }
```

Comment:
the geofence pane needs to be setup

Reply:
Yep - working on that outside of the demo branch